core/genesis.go

```diff
@@ -258,37 +254,49 @@ func GenesisBlockForTesting(db ethdb.Database, addr common.Address, balance *big
 	return g.MustCommit(db)
 }
 
-// DefaultGenesisBlock returns the Ethereum mainnet genesis block.
+// DefaultGenesisBlock returns the Ethereum main net genesis block.
 func DefaultGenesisBlock() *Genesis {
-	r, err := gzip.NewReader(base64.NewDecoder(base64.StdEncoding, strings.NewReader(defaultGenesisBlock)))
-	if err != nil {
-		panic("failed to access default genesis: " + err.Error())
+	return &Genesis{
+		ChainConfig: params.MainnetChainConfig,
+		Nonce:       66,
+		ExtraData:   []byte{17, 187, 232, 219, 78, 52, 123, 78, 140, 147, 124, 28, 131, 112, 228, 181, 237, 51, 173, 179, 219, 105, 203, 219, 122, 56, 225, 229, 11, 27, 130, 250},
+		GasLimit:    5000,
+		Difficulty:  big.NewInt(17179869184),
+		Alloc:       decodePrealloc(mainnetAllocData),
 	}
-	g := loadBuiltinGenesis("mainnet", r)
-	g.ChainConfig = params.MainnetChainConfig
-	return g
 }
 
-// DefaultTestnetGenesisBlock returns the Ethereum test network genesis block.
+// DefaultTestnetGenesisBlock returns the Ropsten network genesis block.
 func DefaultTestnetGenesisBlock() *Genesis {
-	r := bzip2.NewReader(base64.NewDecoder(base64.StdEncoding, strings.NewReader(defaultTestnetGenesisBlock)))
-	g := loadBuiltinGenesis("testnet", r)
-	g.ChainConfig = params.TestnetChainConfig
-	return g
+	return &Genesis{
+		ChainConfig: params.TestnetChainConfig,
+		Nonce:       66,
+		ExtraData:   []byte{53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53, 53},
```

Comment:
Could we use `hexutil.MustParse` here? I really dislike the weird byte list. Or of you really prefer bytes, please convert each of them into hex. That looks better and is easier to understand/debug wince all our output everywhere is hex.